### DISCUSSION TITLE
UX Issues in DOCS page of github rest api

### DISCUSSION THREAD

**Discussion by rehan-ankalgi-7t2** [community/community#40691] [Accessibility] [answered]
### Select Topic Area

Bug

### Body

i headed on to the github rest api docs page to get an overview of the api, i noticed some user experience issue in the UI.

➡ version dropdown has contrast issues, the text is visible only when hovered on
➡ tab labels are not clearly visible
➡ side links below "in this article" are not visible, samilar issue with the sidebar.. when hovered upon it completely hides out the text, making it hard to interpret.

please fix these usability issues!
![Screenshot (145)](https://user-images.githubusercontent.com/58500338/205442703-58753c35-4b9e-463f-b10d-ea40d95ede2e.png)
![Screenshot (143)](https://user-images.githubusercontent.com/58500338/205442707-feb357ae-19fe-4a0e-b97d-ad9b2b7206a1.png)
![Screenshot (144)](https://user-images.githubusercontent.com/58500338/205442709-1b9ac207-a3b0-4778-b1f3-99362df2c554.png)

**Comment by ossanna16** [comment, marked as answer]
Hi there @rehan-ankalgi-7t2 and welcome to our community! Thank you for reporting this!

**Comment by ghostinhershell** [comment]
Hey @rehan-ankalgi-7t2, I appreciate you reaching out about this issue.

I checked out the same page on my end and I'm not quite seeing the same thing. Here's a screenshot of what I'm seeing on my end:
<img width="1511" alt="Screenshot 2022-12-21 at 5 19 47 PM" src="https://user-images.githubusercontent.com/14021469/209014434-d42c0c6a-87f9-4c23-a354-711e17472524.png">

Are you still seeing those contrast issues on your end? If so, can you try viewing the page again [Incognito Mode](https://support.google.com/chrome/answer/95464?hl=en&co=GENIE.Platform%3DDesktop&oco=0) (or your browser's equivalent of that) and see if that changes anything for you? If the issue is resolved in Incognito Mode, clearing your browser's cache and cookies should ultimately do the trick.
